index.html

```diff
@@ -93,14 +94,14 @@ <h2>Heading</h2>
 
         <script src="js/main.js"></script>
 
-        <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
+        <!-- Google Analytics: change UA-113955222-1 to be your site's ID. -->
         <script>
             (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
             function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
             e=o.createElement(i);r=o.getElementsByTagName(i)[0];
             e.src='//www.google-analytics.com/analytics.js';
             r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
```

Comment:
On my local environment, I'm getting an error saying "not allowed to load local resource". Which after some investigation looks like it may have to do with how I opened the file on the index.html file on my local dev environment. Could be worth following up as we continue to work on this code.

Reply:
![image](https://user-images.githubusercontent.com/26756579/36793059-311fe1ee-1c51-11e8-8770-bee0d1501277.png)

I think I am in the right pull request, but I'm wondering if this JS function is the most up to date. 

The one I have in my Analytics Accounts is this: 
![image](https://user-images.githubusercontent.com/26756579/36793422-3bae0c0c-1c52-11e8-9835-9fa55693d007.png)

I'm curious to know the difference between the two and why the JS would work in a static environment.